javascript-functions-practice/get-average-of-three-numbers.js

```diff
@@ -0,0 +1,5 @@
+/* exported getAverageOfThreeNumbers */
+function getAverageOfThreeNumbers(x, y, z) {
+  var AverageOfThreeNumbers = (x + y + z) / 3;
```

Comment:
Make sure to avoid capital letters for the first letter of a variable unless you are defining a class or special const variable